resource-watch/resource-watch
# add coral and air quality topic links

## Overview
This PR adds Coral Reefs and Air Quality as Explore topics, with dashboard links to knowledge graph nodes.  

## Testing instructions
Confirm coral reefs and air quality topics appear in the Explore Topics menu, and that search search by topic results point to the corresponding related dashboard. 

## Pivotal task
Provide the link to the task(s), if any.

---

## Checklist before submitting
